system-tests/tests/smoke/cre/environment-capabilities-don.toml

```diff
@@ -64,6 +65,7 @@
     [nodesets.node_specs.node]
       docker_ctx = "../../../.."
       docker_file = "plugins/chainlink.Dockerfile"
+      # image = "chainlink-tmp:latest"
```

Comment:
Remove? Restore?

Reply:
that's on purpose, so that you:
1. know that you can use an image instead
2. easier for you to uncomment that and comment out `docker_ctx` and `docker_file` than typing it